### PR TITLE
Add recent Wasmtime CVEs

### PR DESCRIPTION
**Pull request by fitzgen**

### CODE REVIEW

File: crates/wasmtime/RUSTSEC-0000-0000.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+```toml
+[advisory]
+id = "RUSTSEC-0000-0000"
+package = "wasmtime"
+date = "2021-09-17"
+url = "https://wasmtime.dev"
```
Comment (Shnatsel) [inline crates/wasmtime/RUSTSEC-0000-0000.md:6]: The `url` field should be relevant to the advisory. If there is no e.g. issue report or announcement about this, I suggest dropping the `url` field.

Comment (Shnatsel) [reply crates/wasmtime/RUSTSEC-0000-0000.md:6]: @tarcieri is there a way to attach multiple URLs yet?

Comment (Shnatsel) [reply crates/wasmtime/RUSTSEC-0000-0000.md]: Ah nevermind, it's `references`

**Comment by Shnatsel** [comment]
Looks good. Thanks!

**Comment by fitzgen** [comment]
Thanks for adding the references!

**Pull request merged**